### PR TITLE
harfbuzz: Update to 8.3.0

### PR DESCRIPTION
**Pull request by Chubercik**
https://github.com/harfbuzz/harfbuzz/releases/tag/8.3.0

- Updated `thirdparty/README.md` to be more precise

### CODE REVIEW

**Review by fire — APPROVED** [review]
Seems like a library upgrade and a stable release.

@bruvzg Might want to look, but it seems low risk.

File: thirdparty/README.md [hunk 1]
```diff
@@ -384,6 +384,7 @@ Files extracted from upstream source:
 - From the `src` folder, recursively:
   - All the `.c`, `.cc`, `.h`, `.hh` files
   - Except `main.cc`, `harfbuzz*.cc`, `failing-alloc.c`, `test*.cc`, `hb-wasm*.*`
```
Comment (AThousandShips) [inline thirdparty/README.md:386]:
```suggestion
  - Except `main.cc`, `harfbuzz*.cc`, `failing-alloc.c`, `test*.cc`, `hb-wasm*.*`,
```
Let's just add an Oxford comma just for correctness

**Comment by akien-mga** [comment]
Could you squash the commits? See [PR workflow](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#the-interactive-rebase) for instructions.

**Comment by Chubercik** [comment]
>See [PR workflow](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#the-interactive-rebase) for instructions.

this_isnt_my_first_rodeo_mr_stark

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**